recipes/sparkle/sparkle.recipe

```diff
@@ -0,0 +1,55 @@
+# -*- Mode: Python -*- vi:si:et:sw=4:sts=4:ts=4:syntax=python
+import shutil
+import logging
+from cerbero.build.build import modify_environment
+import cerbero.utils.messages as m
+
+
+class Recipe(recipe.Recipe):
+    name = 'Sparkle'
+    version = '1.16.0'
+    licenses = [License.MIT]
+    btype = BuildType.CUSTOM
+    remotes = {'origin' : "git@github.com:fluendo/SparkleSharp.git"}
+    commit = 'origin/master'
+
+    files_nuget = [
+            ]
+    files_framework = [
+            "frameworks/Sparkle.framework"
+    ]
+
+    def prepare(self):
+        super(Recipe, self).prepare()
+        if self.config.platform != Platform.DARWIN:
+            raise InvalidRecipeError ()
+        self.files_nuget.append('share/nuget/Sparkle.%s.nupkg' % self.version)
+        # ld does not understand -isysroot so use cc instead for linking
+        self.new_env['LD'] = 'cc'
+
+    @modify_environment
+    def compile(self):
+        shell.call('xcodebuild -configuration Release -target Sparkle',
+                os.path.join(self.build_dir, 'Sparkle/3rdparty/Sparkle'))
+        shell.call("xbuild Sparkle.sln /p:Configuration=Release", self.build_dir)
```

Comment:
this recipe only builds on MacOS but just to let you know. I Had problems in windows because the option /p wasn't accepted -_-.Is needed to put the entire word in the options, Ex: /property:Configuration=Release.